src/System.Reflection/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,4 @@
+Compat issues with assembly System.Reflection:
```

Comment:
Looks like this doesn't have a netstandard1.3 build just a netstandard1.7 build. The reason we try to make these target specific is so that we don't end up masking work in other configurations. In this case I suspect that these are missing from CoreCLR and CoreRT so using the common name should be fine because we likely need to fix both. 

Just for context here is the logic where we pick the ApiCompatBaseline file name https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/ApiCompat.targets#L6.